### ISSUE TITLE
expo-linear-gradient colour bug for web in v7.0.0

### ISSUE THREAD

**Issue by amybird** [expo/expo#5771]
### Environment

Expo CLI 3.0.8 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
    IDEs:
      Xcode: 11.0/11M382q - /usr/bin/xcodebuild
    npmPackages:
      @storybook/react-native: 5.2.0-rc.5 => 5.2.0-rc.5 
      expo: ^35.0.0 => 35.0.0 
      react: 16.8.3 => 16.8.3 
      react-native: ^0.59.8 => 0.59.10 
    npmGlobalPackages:
      expo-cli: 3.0.8

Building for iOS & Web

### Steps to Reproduce

Add LinearGradient component to a screen & run expo for web

### Expected Behavior

Should see linear gradient with specified colours

### Actual Behavior

Linear gradient on web seems to pass through incorrect colours to gradient & also makes opacity under 0.1 so it's barely visible. Version 6 worked fine on web.

### Reproducible Demo
```"expo-linear-gradient": "^7.0.0",```
```
<LinearGradient
          colors={['#b7b7b7', '#000']}
          end={[1, 0]}
          start={[0, 1]}
          style={{ height: 4, borderRadius: 3 }}
        />
```

**Comment by bikashsahucbnits** [comment]
After Upgrading to Expo SDK version to 35. The linear gradient does not seem to be worked for my web project.

**Comment by NiciusB** [comment]
It's shifting the rgba values one to the right.
Any other prop than colors doesn't seem to affect anything.
I've looked at the code of [NativeLinearGradient.web.tsx](https://github.com/expo/expo/blob/master/packages/expo-linear-gradient/src/NativeLinearGradient.web.tsx) but haven't found anything

Here's some examples:
`rgba(0, 255, 0, 0)` gets converted to `rgba(0, 0, 255, 0)`
`rgba(0, 0, 255, 0)` gets converted to `rgba(0, 0, 0, 1)`
`rgba(0, 0, 0, 1)` gets converted to `rgba(255, 0, 0, 0)`
`#004000F0` gets converted to `rgba(240, 0, 64, 0)`  = `#F0004000`
`#000050` gets converted to `rgba(255, 0, 0, 0.31)`  = `#FF000050`

**Comment by bpceee** [comment]
Seems fixing [this one]( https://github.com/expo/expo/issues/5239#issuecomment-529756462) introduced another one? ..

**Comment by amybird** [comment]
> After Upgrading to Expo SDK version to 35. The linear gradient does not seem to be worked for my web project.

@NiciusB if you revert expo-linear-gradient to v6.0.0 in your package.json it should work still with Expo v35.

**Comment by EvanBacon** [comment]
https://github.com/expo/expo/pull/5843

**Comment by EvanBacon** [comment]
fix is live in `expo-linear-gradient@7.0.1`

**Comment by amybird** [comment]
Thank you @EvanBacon!
